dashboard/config/routes.rb

```diff
@@ -148,6 +148,7 @@
       collection do
         post 'section_instructors_verified'
         post 'archive_all'
+        get 'retrieve_lessons_for_dropdown/:section_id', to: 'sections#retrieve_lessons_for_dropdown'
```

Comment:
please put this as `get :retrieve_lessons_for_dropdown` under `member` above, and the section id will be provided to you. (it may come through as `id` or `section_id`)